tests/unit/amo/components/TestAMInstallButton.js

```diff
@@ -121,7 +120,7 @@ describe(__filename, () => {
     const button = root.find(Button);
 
     expect(button).toHaveLength(1);
-    expect(button.childAt(1)).toHaveText('Add to Firefox');
+    expect(button.childAt(0)).toHaveText('Add to Firefox');
```

Comment:
Do we still need `childAt`, or can it now just be `children` because we have removed an element?

Reply:
I think it could be `children` actually, I forgot about it.. I tried without `childAt` (at all) and everything broke 😓 I will update with children, thanks